crates/fluvio-sc/src/k8/controllers/managed_connector_deployment.rs

```diff
@@ -225,7 +228,30 @@ impl ManagedConnectorDeploymentController {
         let parameters: Vec<String> = parameters
             .keys()
             .zip(parameters.values())
-            .flat_map(|(key, value)| [format!("--{}={}", key.replace('_', "-"), value)])
+            .flat_map(|(key, values)| match &values.0 {
+                ManageConnectorParameterValueInner::String(value) => {
+                    vec![format!("--{}={}", key.replace('_', "-"), value)]
+                }
+                ManageConnectorParameterValueInner::Vec(values) => {
+                    let mut args = Vec::new();
+                    for value in values.iter() {
+                        args.push(format!("--{}={}", key.replace('_', "-"), value))
+                    }
+                    args
+                }
+                ManageConnectorParameterValueInner::Map(map) => {
+                    let mut args = Vec::new();
+                    for (sub_key, value) in map.iter() {
+                        args.push(format!(
+                            "--{}-{}={}",
```

Comment:
Do we want this or `--{}="{}={}"`?

Reply:
Not sure for 



```yaml
parameters:
  kafka-options:
    linger.ms: 1
```

Do we want --kafka-options=linger.ms:1 or --kafka-option-linger.ms=1

Reply:
I think we want `--kafka-options=linger.ms:1 `

Reply:
oh, I think I chose wrong.